### PR TITLE
[nereids] explode function stats configurable

### PR DESCRIPTION
**Pull request by xzj7019**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xzj7019** [comment]
run buildall

**Comment by xzj7019** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java [hunk 1]
```diff
@@ -957,7 +957,11 @@ private Statistics computeIntersect(SetOperation setOperation) {
 
     private Statistics computeGenerate(Generate generate) {
         Statistics stats = groupExpression.childStatistics(0);
-        double count = stats.getRowCount() * generate.getGeneratorOutput().size() * 5;
+        int statsFactor = ConnectContext.get().getSessionVariable().generateStatsFactor;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java:960]: u could set value checker to session variable in its annotation to ensure it is valid. refer to session variable `queryTimeoutS`

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -738,6 +740,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_BUCKET_SHUFFLE_DOWNGRADE, needForward = true)
     public boolean enableBucketShuffleDownGrade = false;
 
+    @VariableMgr.VarAttr(name = GENERATE_STATS_FACTOR, needForward = true)
+    public int generateStatsFactor = 5;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:744]: add description to it

**Comment by xzj7019** [comment]
run buildall

**Comment by xzj7019** [comment]
run buildall

**Comment by xzj7019** [comment]
run feut

**Review by morrySnow — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]

**Pull request merged**